MANIFEST

```diff
@@ -6347,6 +6347,7 @@ t/op/getppid.t				See if getppid works
 t/op/glob.t				See if <*> works
 t/op/gmagic.t				See if GMAGIC works
 t/op/goto.t				See if goto works
+t/op/goto-amp-name.t			See if goto &NAME works
```

Comment:
Not a great filename/description. The first instance of `goto` in this file is `goto sub { ... }`, which has neither an ampersand nor a name. Maybe `goto-sub.t` would be better?

Reply:
While I'm not wedded to the filename, it didn't come from nowhere.  It is inspired by `perldoc -f goto` 3rd line.

What I'm more concerned about is if I have got the correct selection of tests between the two files.  I believe that once this is applied, we'll be better able to assess the scope of our test coverage of `goto` and be better positioned to work on https://github.com/Perl/perl5/issues/23618.